main/main.cpp

```diff
@@ -88,17 +88,17 @@
 #endif // PHYSICS_2D_DISABLED
 
 // 3D
-#ifndef PHYSICS_3D_DISABLED
-#include "servers/physics_server_3d.h"
-#include "servers/physics_server_3d_dummy.h"
-#endif // PHYSICS_3D_DISABLED
 #ifndef NAVIGATION_3D_DISABLED
 #include "servers/navigation_server_3d.h"
 #include "servers/navigation_server_3d_dummy.h"
 #endif // NAVIGATION_3D_DISABLED
-#ifndef _3D_DISABLED
+#ifndef PHYSICS_3D_DISABLED
```

Comment:
If we're style nitpicking, I would add separation lines between each block so it's more readable.
```suggestion

#ifndef PHYSICS_3D_DISABLED
```